cts-java/src/test/java/org/ga4gh/cts/api/TestData.java

```diff
@@ -30,7 +31,7 @@ private TestData() {
      * The default ID of the dataset that holds the test data.  We use something readable so the
      * meaning is clear, but in reality the value of this is unlikely to be human-readable.
      */
-    public static final String DEFAULT_DATASET_ID = "compliance-dataset";
+    public static final String DEFAULT_DATASET_ID = "WyJicmNhMSJd";
```

Comment:
Setting the ID to the compliance dataset is part of running the CTK. Assuming our identifier scheme within the software is a step away from standardization. There really isn't a reason for this to have happened other than development configuration as far as I can tell.